views/home.ejs

```diff
@@ -0,0 +1,67 @@
+<%- include("components/header.ejs") %>
+
+<!-- TITLE OF THE PAGE -->
+<title>Social Chef - Home </title>
+
+<!-- FONT USED FOR THE PAGE (RETRIVED FROM GOOGLE FONTS) -->
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Heebo&family=Nunito:wght@200&display=swap" rel="stylesheet">
+<script src="https://kit.fontawesome.com/20718ccc33.js" crossorigin="anonymous" defer></script>
+<!-- CSS UTILIZED BY THIS PAGE -->
+<link rel="stylesheet" href="/css/general.css" type="text/css" />
+<link rel="stylesheet" href="/css/home.css" type="text/css" />  
+
+
+</head>
+
+<body>
+
+  <div class="trim"></div>
+  <nav>
+    <section class="nav-header">
+      <h1>Welcome <br> User!</h1>
+    </section>
+
+    <section class="nav-body">
+      <div class="body-row active"> <i class="fa-sharp fa-solid fa-house"></i><a href="">Home</a> </div>
+      <div class="body-row"><i class="fa-solid fa-bookmark"></i><a href="">Saved</a></div>
+      <div class="body-row"><i class="fa-solid fa-gear"></i><a href="">Info</a></div>
+    </section>
+
+    <section class="nav-footer">
+      <div class="body-row"><i class="fa-solid fa-right-from-bracket"></i><a href="">Logout</a></div>
+    </section>
+
+  </nav>
```

Comment:
[IMO] I think you can use this navigation for all pages Home/Saved/Info, so to change this navigation as component is better.